src/crate/theme/rtd/crate/static/js/custom.js

```diff
@@ -163,3 +163,38 @@
     headerSize();
   });
 })(jQuery);
+
+
+// Admonition box for display on documentation of older releases.
+$(document).ready(function () {
+
+  // Guard against non-availability of relevant information, injected from `layout.html`.
+  if (typeof(rtd_versioning) == "undefined") {
+    console.warn("Disabling `old_versions` admonitions, no `rtd_versioning` information found.");
+    return;
+  }
+
+  // Display `rtd_versioning` information.
+  console.info(`current_version: ${rtd_versioning.current_version}`);
+  console.info(`versions: ${rtd_versioning.versions}`);
+  console.info(`old_versions: ${rtd_versioning.old_versions}`);
```

Comment:
shouldn't these be better turned into `debug` messages?

Reply:
Addressed with 8fde78054e2.